### ISSUE TITLE
Controls always have a dome light

### ISSUE THREAD

**Issue by sparr** [CleverRaven/Cataclysm-DDA#11724]
Even on vehicles that have no power system of any sort, you still have the option to turn on the dome light. This should probably get filtered out for things like bicycles and rowboats.

**Comment by DavidKeaton** [comment]
the controls don't know what it's attached to. i can tell you firsthand that the easiest hack to make would be to test for lack of batteries, alternator, and engine, and don't show option. at that point I'd do that for _ALL_ options needing a power source

**Comment by sparr** [comment]
@DavidKeaton that's not a bad idea. the code in question just has a bunch of "if(has_xxx)" checks. Adding has_electrical_system_at_all is viable.

**Comment by DavidKeaton** [comment]
yeah and it pushes descs back based on em. I added dome lights. :-)

**Comment by DavidKeaton** [comment]
whoops deleted comment:
tl;dr I added dome lights, I know what you be talkin' about. :-)

**Comment by drbig** [comment]
Indeed, 'No electricity no dome lights'.

**Comment by DavidKeaton** [comment]
precisely. it honestly wouldn't take but a handful of lines of code to do.

**Comment by KA101** [comment]
> it honestly wouldn't take but a handful of lines of code to do

If you say so.  ;-)

**Comment by DavidKeaton** [comment]
I can put it in my queue if you'd like. ;-P
